### PR TITLE
New blog post about Che deployment on AKS 

### PR DESCRIPTION
**Pull request by karatkep**
Our story about how we installed Eclipse Che on Microsoft Azure, what problems we met and how solved them.

### CODE REVIEW

**Comment by karatkep** [comment]
CC @l0rd to help with review

**Review by sparkoo — APPROVED** [review]
lgtm

**Comment by karatkep** [comment]
We added more details to the blog post. So this is a final version of proposed blog post.

@l0rd , @sparkoo , could you please review one more time? Thanks in advance!

**Review by ibuziuk — APPROVED** [review]
Thank you for the contribution, great content :+1: 
Added the suggestion for the title to make it more straightforward, drop the negative connotation (not work) and the particular versioning detail that is explained in the blog

**Comment by ibuziuk** [comment]
@karatkep could you please update the blog file with the publication date (we could probably target Monday 07.25) and make the file name match the blog's title. Will wait for the final approval from @l0rd next week before merging.

**Comment by karatkep** [comment]
@ibuziuk thank you. Updated according to the recommendations.

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:74]:
```suggestion
where the ID is the Application ID of the AKS instance registered in AAD.
```

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:183]:
```suggestion
After the {prod} App configuration in Azure is completed, the command `chectl server:deploy` can be used to install {prod} on AKS using the `YAML` file above:
```

File: _posts/2022-07-25-installing-eclipse-che-on-aks.adoc
Comment (l0rd) [inline _posts/2022-07-25-installing-eclipse-che-on-aks.adoc:195]:
```suggestion
NOTE: In our case we already configured `cert-manager` and created `domain` according to the link:https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-microsoft-azure/[old (7.41) instructions].
```

**Review by l0rd — COMMENTED** [review]
Thank you @karatkep, this blog post is great. There is a lot of work behind that. I made a few wording tweaks, please go through them and accept or comment.

**Comment by karatkep** [comment]
Thank you @l0rd. Proposed wording tweaks look perfect!

**Review by l0rd — APPROVED** [review]
LGTM

**Comment by l0rd** [comment]
Thank you @karatkep, feel free to [retweet](https://twitter.com/eclipse_che/status/1552008552293736450).

image

**Pull request merged**
